ui/src/components/Publication/SimpleResult/index.tsx

```diff
@@ -114,9 +115,8 @@ const SimpleResult: React.FC<Props> = (props): React.ReactElement => {
                             Status: {Helpers.getPublicationStatusByAuthor(draftVersion, props.user)}
                         </p>
                         {props.user.id !== draftVersion.user.id ? (
-                            draftVersion.coAuthors.some(
-                                (coAuthor) => coAuthor.email === props.user.email && !coAuthor.linkedUser
-                            ) ? (
+                            // Confirmed coauthor.
+                            draftVersion.coAuthors.some((coAuthor) => coAuthor.linkedUser === props.user.id) ? (
```

Comment:
Here I've just changed the condition and swapped around what is either side of the ternary operator. Looks like more changes than it was.